assembl/static2/js/app/components/debate/brightMirror/fictionPreview.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from 'react';
+import { Link } from 'react-router';
+import truncate from 'lodash/truncate';
+
+const TITLE_MAX_CHAR = 60;
+const NAME_MAX_CHAR = 20;
+
+export type FictionPreviewType = {
+  /** Fiction title */
+  title: string,
+  /** Author fullname */
+  authorName: string,
+  /** Creation date */
+  creationDate: string,
+  /** Url to fiction */
+  link: string,
+  /** Background color */
+  color: string
+};
+
+const FictionPreview = (props: FictionPreviewType) => (
```

Comment:
@cedrtang they use destructured props in here: ({ title, authorName, creationDate, link, color }: FictionHeaderProps)